content/zh/docs/reference/kubectl/kubectl.md

```diff
@@ -32,441 +32,443 @@ kubectl [flags]
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">-s, --server string</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      The address and port of the Kubernetes API server
-      -->
-      Kubernetes API 服务器的地址和端口
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--skip-headers</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      If true, avoid header prefixes in the log messages
-      -->
-      设置为 true 则表示跳过在日志消息中出现 header 前缀信息
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--skip-log-headers</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      If true, avoid headers when opening log files
-      -->
-      设置为 true 则表示在打开日志文件时跳过 header 信息
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--stderrthreshold severity&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 2</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      logs at or above this threshold go to stderr
-      -->
-      等于或高于此阈值的日志将输出到标准错误输出（stderr）
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--token string</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      Bearer token for authentication to the API server
-      -->
-      用于对 API 服务器进行身份认证的持有者令牌
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--user string</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      The name of the kubeconfig user to use
-      -->
-      指定使用 kubeconfig 配置文件中的用户名
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--username string</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      Username for basic authentication to the API server
-      -->
-      用于 API 服务器的基本身份验证的用户名
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">-v, --v Level</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      number for the log level verbosity
-      -->
-      指定输出日志的日志详细级别
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--version version[=true]</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      Print version information and quit
-      -->
-      打印 kubectl 版本信息并退出
-      </td>
-    </tr>
-    <tr>
-      <td colspan="2">--vmodule moduleSpec</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      comma-separated list of pattern=N settings for file-filtered logging
-      -->
-      以逗号分隔的 pattern=N 设置列表，用于过滤文件的日志记录
-      </td>
-    </tr>
+
+  <table style="width: 100%; table-layout: fixed;">
```

Comment:
Probably an explanation on what we're trying to fix will be helpful. At least put the explanation in commit message or description on the PR.